data/Sri Guru Granth Sahib Ji/0196.json

```diff
@@ -355,7 +355,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "Meditation, self-discipline, self-control and perfect greatness were obtained when the Merciful Lord,",
+              "translation": "Meditation, self-discipline, self-control and perfect greatness",
```

Comment:
```suggestion
              "translation": "Meditation, self-discipline, self-control and perfect greatness were obtained",
```

Reply:
Should remain as is, `were obtained` comes in the next line.